packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -147,6 +152,33 @@ class SegmentedControl<T> extends StatefulWidget {
   /// ```
   final ValueChanged<T> onValueChanged;
 
+  /// The color used to fill the backgrounds of unselected widgets and as the
+  /// text color of the selected widget.
+  ///
+  /// If this attribute is null, this color will be
```

Comment:
This is not true. If this parameter is null, it will be null :)

If it's unspecified, it defaults to CupertinoColors.white.